src/main/resources/assets/complicated_bees/lang/en_us.json

```diff
@@ -217,19 +218,24 @@
   "gene.complicated_bees.species_label": "Species",
   "gene.complicated_bees.hybrid": "Hybrid",
   "gene.complicated_bees.humidity_label": "Humidity",
+  "gene.complicated_bees.humidity_label.short": "H: %1$s / %2$s",
   "gene.complicated_bees.temperature_label": "Temperature",
-  "gene.complicated_bees.lifespan_label": "Life",
+  "gene.complicated_bees.temperature_label.short": "T: %1$s / %2$s",
+  "gene.complicated_bees.lifespan_label": "%s Life",
```

Comment:
This label is used for frames, where the `%s` is not used and causes a visual bug.
This is why the other labels are split into `*_label` and `*_label.short`.